.github/workflows/ci.yaml

```diff
@@ -22,8 +22,6 @@ jobs:
   build-windows-pr: # job-name skip-master skip-stable
     runs-on: windows-latest
     if: ${{ contains('["pull_request", "merge_group"]', github.event_name) }} # skip-master skip-stable
-    env:
```

Comment:
This is for testing only. Eventually we'd still like to keep this to prevent our users from having to install a CRT to be able to run our Windows MSVC binary.